handlers/secrets.go

```diff
@@ -82,7 +82,13 @@ func UpdateSecrets(request requests.CreateFunctionRequest, deployment *v1beta1.D
 			},
 		}
 
-		deployment.Spec.Template.Spec.Volumes = append(deployment.Spec.Template.Spec.Volumes, projectedSecrets)
+		existingVolumes := deployment.Spec.Template.Spec.Volumes
+		for i, v := range existingVolumes {
```

Comment:
Could you break out a method?

Reply:
Sorry if I am being dense, but I am not sure I entirely follow.  Which part of the `UpdateSecrets` function should be broken out further? Is there a portion of the code that should be reusable somewhere else?